contract-ui/tabs/overview/cards/custom-contract-code.tsx

```diff
@@ -228,9 +229,7 @@ const InteractiveAbiFunction: React.FC<InteractiveAbiFunctionProps> = ({
     data,
     error,
     isLoading: mutationLoading,
-  } = useMutation(async (params: unknown[] = []) =>
-    abiFunction ? await contract?.call(abiFunction.name, ...params) : undefined,
-  );
+  } = useMutation(async (params: unknown[] = []) => contractCall(params));
```

Comment:
can we move this below the definition of `contractCall()` (I know it may work without doing that but it hurts my brain to use a function above where we declare it)